src/unistd.rs

```diff
@@ -268,14 +271,13 @@ pub fn setsid() -> Result<Pid> {
 /// [getsid(2)](http://pubs.opengroup.org/onlinepubs/9699919799/functions/getsid.html).
 ///
 /// Obtain the process group ID of the process that is the session leader of the process specified
-/// by pid. If pid is zero, it specifies the calling process.
+/// by pid. If passed None or pid is or zero, it specifies the calling process.
```

Comment:
Same as the above, do we really want to allow `0` as a valid input. I'd argue that we shouldn't.